src/app/ideas/IdeaDetails.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import Tag from './Tag';
+
+library.add(faTimes);
+
+const IdeaContainer = styled.section`
+  display: grid;
+  grid-template-rows: 30px auto;
+  padding: 20px;
+  margin: 10px;
+  overflow: scroll;
+  background: lightslategray;
+`;
+
+const IdeaTitle = styled.h2`
+  font-size: 1.2rem;
+`;
+
+const IdeaText = styled.p`
+  font-size: 1rem;
+  line-height: 1.8rem;
+`;
+
+const StyledNavLink = styled(NavLink)`
+  display: grid;
```

Comment:
why do you use `display: grid` but no column or row definition? maybe in this case a flexbox would make more sense?

Reply:
I don't need to. I put it in the grid so that it is in the grid of the `IdeaContainer`, which has a 30px row at the top for the `X`icon.